labelbox/data/annotation_types/data/tiled_image.py

```diff
@@ -0,0 +1,482 @@
+import math
+import logging
+from enum import Enum
+from typing import Optional, List, Tuple
+from concurrent.futures import ThreadPoolExecutor
+from io import BytesIO
+
+import requests
+import numpy as np
+
+from retry import retry
+from PIL import Image
+from pyproj import Transformer
+from pygeotile.point import Point as PygeoPoint
+from pydantic import BaseModel, validator
+from pydantic.class_validators import root_validator
```

Comment:
Make sure to add any dependencies to the setup.py 
https://github.com/Labelbox/labelbox-python/blob/83aebf1124c1111156997697f7cf6ed217fefdd2/setup.py#L35